staging/src/k8s.io/kubectl/pkg/polymorphichelpers/portsforobject_test.go

```diff
@@ -113,28 +194,31 @@ func TestPortsForObject(t *testing.T) {
 					},
 				},
 			},
+			expected: []string{"101"},
 		},
 		{
 			object:    &corev1.Node{},
 			expectErr: true,
 		},
 	}
-	expectedPorts := []string{"101"}
 
 	for _, test := range tests {
-		actual, err := portsForObject(test.object)
-		if test.expectErr {
-			if err == nil {
-				t.Error("unexpected non-error")
+		t.Run(test.name, func(t *testing.T) {
```

Comment:
I understand the purpose in here. But considering that there are several changes for the fix, it would be better not touch unit tests(unless it is essential) to better verify the actual changes.

Reply:
Thank you. Let me withdraw this change. Can I add a PR to complete the change here?